### PR TITLE
fix cugraph notebooks

### PR DESCRIPTION
**Pull request by AyodeAwe**
Certain cugraph notebooks are excluded from testing as is implemented [here](https://github.com/rapidsai/cugraph/issues/2622). This PR updates this repository to reflect that requirement.

### CODE REVIEW

File: context/test_notebooks.py [hunk 1]
```diff
@@ -43,7 +44,7 @@ def get_notebooks(directory: str) -> Iterable[str]:
                     any(match_ignored_file in file for match_ignored_file in ignored_filenames)
                 )
                 and "MNMG" not in file.upper()
-                and file not in ignored_notebooks
+                and not (any(file in ignored_notebook for ignored_notebook in ignored_notebooks))
```
Comment (bdice) [inline context/test_notebooks.py:47]: Typically Python isn’t written with parentheses around the not operator.
```suggestion
                and not any(file in ignored_notebook for ignored_notebook in ignored_notebooks)
```

File: context/test_notebooks.py [hunk 2]
```diff
@@ -43,7 +44,7 @@ def get_notebooks(directory: str) -> Iterable[str]:
                     any(match_ignored_file in file for match_ignored_file in ignored_filenames)
                 )
                 and "MNMG" not in file.upper()
-                and file not in ignored_notebooks
+                and not any(file in ignored_notebook for ignored_notebook in ignored_notebooks)
```
Comment (jakirkham) [inline context/test_notebooks.py:47]: Would be tempted to rewrite this `if` with `set` operations above on `files`. That would simplify the code improving readability and cutting down on unneeded iteration

**Comment by bdice** [comment]
Do we think these errors will be fixed by https://github.com/conda-forge/ipython-feedstock/pull/210? Is there more work needed after that PR?

**Review by raydouglass — APPROVED** [review]

**Pull request merged**